specification/applicationinsights/resource-manager/readme.md

```diff
@@ -324,6 +324,28 @@ input-file:
 - Microsoft.Insights/preview/2020-03-01-preview/componentLinkedStorageAccounts_API.json
 ```
 
+### Tag: package-2020-04
```

Comment:
what's this tag used for? If it's only for PS, we may name it as a specific name.

Reply:
yes, it's for PS to include APIs 2020-03-01-preview and 2020-02-02-preview.

what kind of name can I use? can you give some examples?

Reply:
like `package-2020-04-powershell`? or add the information into the description.

Reply:
description added